build.gradle

```diff
@@ -264,7 +264,7 @@ allprojects {
         appCompatVersion = "1.2.0"
         azureCommunicationCommonVersion = "1.0.1"
         azureCoreVersion = "1.0.0-beta.9"
-        jacksonVersion = "2.12.5" // Do not upgrade to 2.13, as it introduced using an API not available in javax.xml.stream:stax-api:1.0-2
+        jacksonVersion = "2.12.6" // Do not upgrade to 2.13, as it introduced using an API not available in javax.xml.stream:stax-api:1.0-2
```

Comment:
is there a bug tracking the 2.13 issue that can be added to this comment?

Reply:
Yes, I created #1017 and will add it to the comment in our `build.gradle` file.